app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -1,113 +1,133 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
+
   <data>
-    <import type="android.text.InputType"/>
+
+    <import type="android.text.InputType" />
+
     <variable
       name="viewModel"
       type="org.oppia.app.profile.PinPasswordViewModel" />
   </data>
+
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/pinPasswordBackground">
+    android:background="@color/oppiaLightGreen">
+
     <TextView
-      android:id="@+id/hello_text"
+      android:id="@+id/hello_text_view"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:lines="1"
+      android:layout_marginStart="28dp"
+      android:layout_marginTop="96dp"
+      android:layout_marginEnd="28dp"
+      android:fontFamily="sans-serif"
       android:gravity="center_horizontal"
+      android:lines="1"
       android:text="@{String.format(@string/pin_password_hello, viewModel.profile.name)}"
-      android:textColor="@color/black"
-      android:textSize="28sp"
-      android:layout_marginTop="40dp"
-      app:layout_constraintTop_toTopOf="parent"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="24sp"
+      app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"/>
+      app:layout_constraintTop_toTopOf="parent" />
+
     <TextView
-      android:id="@+id/enter_text"
+      android:id="@+id/enter_text_view"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
+      android:layout_marginStart="60dp"
+      android:layout_marginTop="8dp"
+      android:layout_marginEnd="60dp"
+      android:fontFamily="sans-serif"
       android:gravity="center_horizontal"
       android:text="@{viewModel.profile.isAdmin ? @string/pin_password_admin_enter : @string/pin_password_user_enter}"
-      android:textColor="@color/black"
-      android:textSize="24sp"
-      android:layout_marginTop="12dp"
-      android:paddingBottom="12dp"
-      app:layout_constraintTop_toBottomOf="@+id/hello_text"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="18sp"
+      app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"/>
+      app:layout_constraintTop_toBottomOf="@+id/hello_text_view" />
+
     <TextView
-      android:id="@+id/error_text"
+      android:id="@+id/error_text_view"
```

Comment:
nit change please make the commonly used widget naming with file name to avoid renaming issues.

Reply:
done.